frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignContent/useBuilderFields.tsx

```diff
@@ -19,10 +21,12 @@ import {
   useFieldBuilderStore,
 } from '../useFieldBuilderStore'
 
-const getFormFieldsWhileCreating = (
+setAutoFreeze(false)
```

Comment:
should we add a comment here so future devs know what this is doing?

Reply:
ideally would like to not enable this flag by fixing the issue that necessitates it, but for now a comment could be good